vsphere/tests/legacy/utils.py

```diff
@@ -142,3 +142,25 @@ def get_mocked_server():
     server_mock = MagicMock()
     server_mock.configure_mock(**{'RetrieveContent.return_value': content_mock, 'content': content_mock})
     return server_mock
+
+
+def mock_alarm_event(from_status='green', to_status='red', message='Some error', key=0, created_time=None):
```

Comment:
nit/question: when is the created_time parameter used? Or will it be needed in future tests?

Reply:
@sarah-witt Yes. It's used in tests in other PRs. Example:

https://github.com/DataDog/integrations-core/pull/6656/files#diff-c3a8f84cdfda9967b3caccf00fd9cfa4R23-R25